lib/kube/proxy/sess.go

```diff
@@ -554,7 +554,7 @@ func (s *session) launch(isEphemeralCont bool) error {
 	defer func() {
 		// The closure captures the err variable pointer so that the variable can
```

Comment:
This comment should be updated now, right?

Reply:
done in [e6031e4](https://github.com/gravitational/teleport/pull/42134/commits/e6031e42f0ee370af7f8f4bbd8d8515db0efa99b)